### ISSUE TITLE
Click switch user on lock screen wont work

### ISSUE THREAD

**Issue by jsalatiel** [linuxmint/Cinnamon#5121]
This has happened in three fresh installs of mint 17.3 in three different machines.
If any user locks its screen, and i click switch user to be able to login with out logging out the previous user it wont work. I wont get a login/password prompt, it will just keep asking the current user password. When the current user unlock the screen, the attached  error message will be in its desktop.
![mint](https://cloud.githubusercontent.com/assets/5522389/13554040/1ec7e394-e37c-11e5-955b-5b8c670d3bac.png)

**Comment by jsalatiel** [comment]
I found the source of the problem. Apparently it is not possible to switch user if ipv6 is disabled 
"net.ipv6.conf.all.disable_ipv6=1" on /etc/sysctl.conf.

Why is mdmflexiserver dependent on ipv6 ?

**Comment by abdupattoh** [comment]
I am having this exact same problem, but there is no net.ipv6.conf.all.disable_ipv6=1" in my /etc/sysctl.conf on either of my LinuxMint 18 machines.  

So yes bug, no fix.

**Comment by leigh123linux** [comment]
Duplicate of https://github.com/linuxmint/mdm/issues/178
